plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2Record.groovy

```diff
@@ -30,7 +30,6 @@ class CO2Record extends TraceRecord {
         this.energy = energy
         this.co2e = co2e
         this.time = time
-        this.cpus = cpus
         this.powerdrawCPU = powerdrawCPU
         this.cpuUsage = cpuUsage
```

Comment:
Why is this removed? 
It seems I forgot to add it to `this.store` below, but wouldn't it be needed also for the report table?

Reply:
Ups that was unintentional, I only wanted to remove the duplicated co2e line below, thanks for spotting it!